cdap-ui/app/directives/my-pipeline-summary/my-pipeline-summary-ctrl.js

```diff
@@ -68,14 +68,14 @@ class MyPipelineSummaryCtrl {
     if (nextRunTime && nextRunTime.length) {
       this.nextRunTime = nextRunTime[0].time? nextRunTime[0].time: null;
     } else {
-      this.nextRunTime = 'N/A';
+      this.nextRunTime = '--';
```

Comment:
If this is suppose to be an en- or em-dash, is it possible to have the correct character? "–" or "—" or on a Mac, [shift-]option-dash.

Though since this being used as a test (in the summary.html), would it not make more sense to have the value as an empty string (as strings are being used here) and then have whatever is displaying this value, at that point, change it from an empty string to either an appropriate message ("Not available", "Not applicable", perhaps as a mouse-over) or an en- or em-dash ("—", or the HTML entity, "&emdash;")…